platform/platform_tests/distance_tests.cpp

```diff
@@ -45,7 +45,7 @@ UNIT_TEST(Distance_CreateFormatted)
     TEST_EQUAL(d.GetUnits(), Distance::Units::Meters, ());
     TEST_ALMOST_EQUAL_ULPS(d.GetDistance(), 100.0, ());
     TEST_EQUAL(d.GetDistanceString(), "100", ());
-    TEST_EQUAL(d.ToString(), "100 m", ());
+    TEST_EQUAL(d.ToString(), "100" + kNonBreakingSpace + "m", ());
```

Comment:
`"100" + kNonBreakingSpace + "m"`

I apologize for raising what might seem like trivial matters, but I believe it would enhance code readability to establish a distinct function. This way, we can encapsulate the use of kNonBreakingSpace and prevent its repetition on every line.

Reply:
Hi AndrewShkrob! Please do not hesitate to make any comment about my code. You're most probably used to the coding standards/style in OM.

I've added the `static` function `Distance::ToString()` which encapsulates the usage of `kNonBreakingSpace`.

Please have a look at the code that I've just pushed and confirm that this approach is Ok.

Reply:
Certainly, there's no need for such a function inside the Distance class. Since it will solely be utilized in unit tests, I suggest moving it to `platform/platform_tests/distance_tests.cpp`.

Moreover, I concur that `ToString` might not be the most suitable name. How about renaming it to `MakeDistanceStr` or a similar variant?

Reply:
Ok!! I've moved the function (now called `MakeDistanceStr()`) to `platform/platform_tests/distance_tests.cpp`.